### PR TITLE
Kill bridge followup

### PR DESCRIPTION
**Pull request by martykan**
## Description

Several changes to continue the process of removing references to Trezor Bridge

### CODE REVIEW

File: packages/suite/src/components/suite/DeviceInvalidModeLayout.tsx [hunk 1]
```diff
@@ -1,83 +0,0 @@
-import { ReactNode } from 'react';
```
Comment (mroz22) [inline packages/suite/src/components/suite/DeviceInvalidModeLayout.tsx:1]: these files here were never used / duplicates?

File: packages/suite/src/views/suite/device-unknown/index.tsx [hunk 2]
```diff
@@ -1,7 +0,0 @@
-// TODO: remove whole file, replaced by @suite-components/PrerequisitesGuide/components/DeviceUnknown
```
Comment (mroz22) [inline packages/suite/src/views/suite/device-unknown/index.tsx:1]: ahh, I see, there is even a comment saying that

Comment (martykan) [reply packages/suite/src/views/suite/device-unknown/index.tsx:1]: Yeah 3 years ago 😄

File: packages/suite/src/hooks/suite/useOpenSuiteDesktop.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import { SUITE_BRIDGE_DEEPLINK, SUITE_URL } from '@trezor/urls';
+import { isWebUsb } from 'src/utils/suite/transport';
+import { useSelector } from 'src/hooks/suite';
+
+export const useOpenSuiteDesktop = () => {
+    const transport = useSelector(state => state.suite.transport);
+    const windowFocused = useWindowFocus();
+    const handleOpenSuite = () => {
+        if (isWebUsb(transport)) {
+            TrezorConnect.disableWebUSB();
```
Comment (mroz22) [inline packages/suite/src/hooks/suite/useOpenSuiteDesktop.ts:12]: hmm I am brainstorming (not related to this code here). The problem: https://github.com/trezor/trezor-suite/issues/13179

We at least used to have manual solution - disable webusb button. It was not perfect and I think we should address it now. The right solution would be to disable webusb somewhere under the hood based on either bridge polling, or when a connect call ends with (specific) error.

**Review by mroz22 — APPROVED** [review]

**Pull request merged**